ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/DynamoDBKeyValue.java

```diff
@@ -287,6 +319,131 @@ private void initializeTable() {
         } catch (InterruptedException e) {
             logger.error("Table interrupted exception: " + e);
         }
+
+        // if autoscaling was disabled, we are finished
+        if (!config.getAutoscaling()) {
+            return;
+        }
+
+        // configure autoscaling - from aws docs
+        // https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/AutoScaling.HowTo.SDK.html
+        final String resourceID = "table/" + config.getTableName();
+        final String accountId = securityTokenService.getCallerIdentity(new GetCallerIdentityRequest()).getAccount();
+        final String serviceRoleArn = "arn:aws:iam::" + accountId + ":role/aws-service-role/dynamodb.application-autoscaling.amazonaws.com/AWSServiceRoleForApplicationAutoScaling_DynamoDBTable";
+        final DescribeLimitsResult limits = client.describeLimits(new DescribeLimitsRequest());
+        final Integer targetWriteUtilization = Integer.parseInt(config.getTargetWriteUtilization());
+        final Integer targetReadUtilization = Integer.parseInt(config.getTargetReadUtilization());
+        createAndVerifyScalableTargetAndScalingPolicy(serviceRoleArn,
+                resourceID,
+                ScalableDimension.DynamodbTableWriteCapacityUnits,
+                MetricType.DynamoDBWriteCapacityUtilization,
+                writeCapacityUnits,
+                limits.getTableMaxWriteCapacityUnits().intValue(),
+                targetWriteUtilization);
+        createAndVerifyScalableTargetAndScalingPolicy(serviceRoleArn,
```

Comment:
Are we calling `createAndVerifyScalableTargetAndScalingPolicy` twice?

Reply:
yes, once for the __read__ utilization/capacity target and once for the __write__ utilization/capacity target

Reply:
Good point. I missed the difference in the arguments. It might worth adding a comment above for people who read the code.